sdk/core/ci.yml

```diff
@@ -37,8 +37,7 @@ pr:
 stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
     parameters:
-      ServiceDirectory: "*"
-      DocDirectory: "core"
```

Comment:
Don't you need to also remove the usages of the `DocDirectory`? Or is that used somewhere else?

Reply:
From a quick look (https://github.com/Azure/azure-sdk-for-js/search?q=DocDirectory&unscoped_q=DocDirectory) it doesn't seem like it is used anywhere else so please also remove it from the templates.

Reply:
removed